examples/fundamentals__fixtures/cypress/integration/multiple-fixtures-spec.js

```diff
@@ -0,0 +1,30 @@
+/// <reference types="cypress" />
+describe('Loading multiple fixtures', () => {
+  it('loads fixtures one by one', () => {
+    // pyramid of doom of fixtures
+    cy.fixture('city').then(city => {
+      cy.fixture('country').then(country => {
+        // thanks to JavaScript closures we have access to both fixtures
+        expect({ city, country }).to.deep.equal({
+          city: { name: 'Atlanta' },
+          country: { name: 'United States' }
+        })
+      })
+    })
+  })
+
+  it.skip('loads fixtures using Cypress.Promise.all - not recommended due to race condition', () => {
```

Comment:
I think this feels a bit weird - to add an example of an antipattern into the example recipes.

Reply:
sure I can remove it - but people will think `why can't I use Promise.all` ...

Reply:
> sure I can remove it - but people will think `why can't I use Promise.all` ...

That's exactly how I reached this notes